language/stdlib/modules/LibraConfig.move

```diff
@@ -192,10 +192,10 @@ module LibraConfig {
 
     spec module {
 
-        // Verification is disabled because of a false error in signer, called
-        // from offer.  There are other problems that may be genuine, but we have to
-        // debug the previous first.
-        pragma verify = false;
+        /// Specifications of LibraConfig are very incomplete.  There are just a few
+        /// definitions that are used by RegisteredCurrencies
+
+        pragma verify = true;
```

Comment:
I can't comment on line 208 so I'm commenting here. Line 208 should probably be changed to `spec_is_published  is the spec version of is_published<Config>`.